en/api/index.md

```diff
@@ -24,4 +26,4 @@ export default {
 }
 ```
 
-<div class="Alert Alert--orange">You do **NOT** have access of the component instance through `this` inside `asyncData` because it is called **before initiating** the component.</div>
+<p class="Alert Alert--orange"><b>Warning:</b> You **don't** have access of the component instance through `this` inside `asyncData` because it is called **before initiating** the component.</p>
```

Comment:
@manniL is there different rendering for `<b>` and `**`?

Reply:
I don't think so! (example: https://nuxtjs.org/faq/github-pages#how-to-deploy-on-github-pages-) 

Good point.
Still need to find out why markdown in HTML tags isn't properly resolved in my local setup but on the live page (cc @Atinux :pray:)

Reply:
Good question, I will switch to another Markdown to HTML parser soon, you can keep HTML like you did.